src/NuGet.Clients/NuGet.SolutionRestoreManager/RestoreOperationLogger.cs

```diff
@@ -213,12 +213,12 @@ private void HandleErrorsAndWarnings(ILogMessage logMessage)
 
         private static bool ShouldReportProgress(ILogMessage logMessage)
         {
-            // Only show messages with VerbosityLevel.Normal. That is, info messages only.
+            // Only show messages with VerbosityLevel.Minimal.
             // Do not show errors, warnings, verbose or debug messages on the progress dialog
             // Avoid showing indented messages, these are typically not useful for the progress dialog since
             // they are missing the context of the parent text above it
             return RestoreOperationProgressUI.Current != null
-                && GetMSBuildLevel(logMessage.Level) == MSBuildVerbosityLevel.Normal
+                && GetMSBuildLevel(logMessage.Level) == MSBuildVerbosityLevel.Minimal
                 && logMessage.Message.Length == logMessage.Message.TrimStart().Length;
```

Comment:
I know this isn't part of your change but this line:
```c#
logMessage.Message.Length == logMessage.Message.TrimStart().Length
```
Seems suboptimal and runs for every single message that gets logged.  I guess you're logging way less now but maybe we should make this better
```c#
logMessage.Message.Length > 0 && !Char.IsWhiteSpace(logMessage.Message, 1)
```

It just raised a red flag...

Reply:
That's a great point. 

I'll check how `TrimStart` works and fix it.

Reply:
TrimStart is optimized. 

```cs

CheckIfStrigHasWhitespace("stuff");
CheckIfStrigHasWhitespace("  stuff");

static void CheckIfStrigHasWhitespace(string stuff)
{
    var stuffString = stuff.TrimStart();

    Console.WriteLine(stuff.Equals(stuffString));
    Console.WriteLine(Object.ReferenceEquals(stuff, stuffString));
}
``` 

prints: 

true
true
false
false

Your suggestion is more efficient in the indented case though, so it's still worth making it.